scripts/generate-identifier-data.js

```diff
@@ -44,13 +44,26 @@ var generateES5Data = function() { // ES 5.1
 
 var fs = require('fs');
 var writeFile = function(fileName, data) {
-  fs.writeFileSync(
+  var old=data[0], n=1, s="[", c=old;
+  for (var i=1;i<data.length;i++){ // encode continuous sequences of numbers into elements of array [number, sequence length]
```

Comment:
It would be good to use descriptive variable names, not just n/s/c.
It would also be good to follow the style from the rest of the project - it isn't enforced in this file, but may be in the future, e.g. missing spaces, like between `n=1`, tab size should be 2 spaces.